tests/js/setup.js

```diff
@@ -86,6 +79,26 @@ window.scrollTo = jest.fn();
 // emotion context broadcast
 const broadcast = createBroadcast(theme);
 
+// Dynamically load fixtures
+const fixturesPath = `${__dirname}/fixtures`;
+const modules = fs
+  .readdirSync(fixturesPath)
+  .map(filename => require(`${fixturesPath}/${filename}`));
```

Comment:
Do we need to support fixtures in subdirectories?

Reply:
I think it's ok for now - but we should followup to have it more dev-ex friendly